cli/ssh.go

```diff
@@ -284,13 +291,21 @@ func (r *RootCmd) ssh() *serpent.Command {
 					return err
 				}
 
+				var errCh <-chan error
+				if networkInfoDir != "" {
+					errCh, err = setStatsCallback(ctx, conn, logger, networkInfoDir, networkInfoInterval)
```

Comment:
Hey there! Warming up to this part of the code so bear with me haha. I can't tell what's different about this call to setStatsCallback vs. the next call a little ways down. Seems like the PIDs would be the same (since it's the parent) so we'd be writing to the same file?

Reply:
Oh I see the return statement now 💀

Reply:
Oh the `stdio` code block has a `return` at the end of it. I missed that as well.

Not important now, but there is probably a refactor to make that code path more obviously end. Maybe extracting the if statement to a function so it's like:
```golang
if stdio {
   return handleStdio()
}
```